deepinv/physics/forward.py

```diff
@@ -616,6 +595,127 @@ def A_dagger(
         )
 
 
+class ComposedPhysics(Physics):
+    r"""
+    Composes multiple physics operators into a single operator.
+
+    The measurements produced by the resulting model are defined as
+
+    .. math::
+
+        A(x) = N_k(A_k(\dots(A_1(x))))
```

Comment:
```suggestion
        A(x) = N_k(A_k(\dots A_2(A_1(x))))
```

I would add this to make clear that only the last noise is applied